src/gwt/src/org/rstudio/studio/client/workbench/prefs/PrefsConstants_fr.properties

```diff
@@ -487,7 +487,7 @@ editingTabWidthLabel=Largeur de la tabulation
 editingAutoDetectIndentationLabel=Détection automatique de l''indentation dans les fichiers
 editingAutoDetectIndentationDesc=Lorsque cette option est activée, l''indentation des documents ne faisant pas partie d''un projet RStudio sera automatiquement détectée.
 editingInsertMatchingLabel=Insertion automatique des parenthèses et des crochets correspondants
-editingInsertNativePipeOperatorLabel=Utiliser l''opérateur pipe natif de R, |>
+editingInsertNativePipeOperatorLabel=Utiliser l''opérateur pipe natif de R, |> PLEASE UPDATE
```

Comment:
This translation is missing the parenthetical (Requires R 4.1+) 
Could someone please add?